lib/src/core/string_loader.dart

```diff
@@ -0,0 +1,22 @@
+import 'dart:io';
+import 'package:flutter/services.dart';
+///
+abstract class StringLoader {
```

Comment:
how about rename to JsonFile or JsonData

Reply:
Done